SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DebugInterface/SpatialDebugInterfaceTest.cpp

```diff
@@ -395,3 +395,20 @@ void ATestDebugInterface::PrepareTest()
 		},
 		5.0f);
 }
+
+USpatialDebugInterfaceMap::USpatialDebugInterfaceMap()
+	: UGeneratedTestMap(EMapCategory::CI_PREMERGE, TEXT("SpatialDebugInterfaceMap"))
```

Comment:
not spatial only? Would have thought this would only work with spatial

Reply:
Oh yeah, good catch!
Although we may need to move it to NO_CI for now, looks like it's flaking
https://buildkite.com/improbable/unrealgdk-premerge/builds/23163#70f26ec7-e3f1-48f0-91aa-12191008d197
```
[2021.03.05-13.22.46:925][961]LogSpatialGDKFunctionalTests: Display: >> Starting Step Check new actors interest and delegation on: [Server:1], [Server:2]
--
  | [2021.03.05-13.22.47:941][ 22]LogSpatialGDKFunctionalTests: Display: [Server:2] finished Step
  | [2021.03.05-13.22.51:924][261]LogSpatialGDKFunctionalTests: Display: Test SpatialDebugInterfaceTest_0 finished! Result: EFunctionalTestResult::Failed ; Message: Step time limit reached
  | [2021.03.05-13.22.51:958][263]LogFunctionalTest: Error: SpatialDebugInterfaceMap SpatialDebugInterfaceTest_0 Failed: "Step time limit reached"
  | [2021.03.05-13.22.52:008][266]LogAutomationController: Error: Test Completed. Result={Failed} Name={SpatialDebugInterfaceTest} Path={Project.Functional Tests./Game/Intermediate/Maps/CI_Premerge/SpatialDebugInterfaceMap.SpatialDebugInterfaceTest}
  | [2021.03.05-13.22.52:008][266]LogAutomationController: Error: SpatialDebugInterfaceMap SpatialDebugInterfaceTest_0 Failed: "Step time limit reached" [C:\b\1\unrealengine-premerge\Engine\Source\Runtime\Core\Private\Logging\LogMacros.cpp(92)]

```

Reply:
oh yeah, btw, we should run all the tests in the new suite a few times to make sure we are not introducing too much flakery

Reply:
All of the tests added in this PR have been tested > 10 times in the editor, but this one seems to only timeout in CI (tested in the editor > 30 times).